commands/ConfigCommand.js

```diff
@@ -107,11 +107,12 @@ ConfigCommand.prototype = extend(BaseCommand.prototype, {
 	changeSetting: function (group, name, value) {
 		settings.override(group, name, value);
 	},
-	
+
 	identifyServer: function () {
 		console.log("Current profile: " + settings.profile);
 		console.log("Using API: " + settings.apiUrl);
 		console.log("Access token: " +  settings.access_token);
+    console.log("Default compile target: " +  settings.defaultCompileDevice);
```

Comment:
@brycekahle i have added this line for a complete PR but somehow Atom doesn't follow the tab/indent you guys are using. Any hints?